dev.k8slens.OpenLens.yml

```diff
@@ -38,6 +38,30 @@ modules:
           project-id: 5097
           url-template: https://github.com/libvips/libvips/archive/refs/tags/v$version.tar.gz
 
+  - name: kubelogin
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 bin/*/kubelogin "${FLATPAK_DEST}/bin/kubelogin"
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/Azure/kubelogin/releases/download/v0.0.28/kubelogin-linux-amd64.zip
+        sha256: c854ae5d704b645aba0e90351ced1add1cd2a3bd13a392883053a371f60b6a4a
+        x-checker-data:
+          type: anitya
+          project-id: 341508
+          url-template: https://github.com/Azure/kubelogin/releases/download/$version/kubelogin-linux-amd64.zip
+      - type: archive
+        only-arches:
+          - arm64
+        url: https://github.com/Azure/kubelogin/releases/download/v0.0.28/kubelogin-linux-amd64.zip
```

Comment:
I think this url should be:
https://github.com/Azure/kubelogin/releases/download/v0.0.28/kubelogin-linux-arm64.zip